daemon/Cargo.toml

```diff
@@ -45,6 +32,13 @@ fern = { version = "0.6.0", optional = true }
 chrono = { version = "0.4.19", default-features = false, features = ["clock"], optional = true }
 pin-project = "1.0"
 uuid = { version = "0.8.2", features = ["v4"] }
+async-io = "1.4.1"
+async-executor = "1.4.1"
+async-net = "1.6.0"
+async-fs = "1.5.0"
+async-lock = { version = "2.1" }
+async-channel = { version = "1.4" }
```

Comment:
I think `async-channel` and `async-lock` can both be dropped because we're using `futures`.